packages/content/src/Application/ContentResolver/Value/ContentView.php

```diff
@@ -217,4 +217,37 @@ public function setReferences(array $references): self
 
         return $this;
     }
+
+    /**
+     * Recursively collect all references from this ContentView and nested ContentViews.
+     * Each reference will have its path set to indicate where it was found.
+     *
+     * @return array<Reference>
+     */
```

Comment:
We maybe at some state move it to a service so I would make it currently `@internal`.

```suggestion
    /**
     * @internal This method can be removed at any time no backwards compatibility promise is given for this.
     *
     * Recursively collect all references from this ContentView and nested ContentViews.
     * Each reference will have its path set to indicate where it was found.
     *
     * @return array<Reference>
     */
```